api/ws/rpc/index.js

```diff
@@ -2,6 +2,7 @@
  * @namespace rpc
  * @memberof api.ws
  * @see Parent: {@link api.ws}
+ * @property {module:api/ws/rpc/failure_codes} Failure_codes
```

Comment:
Yes, for example in the db-PR.
We have to include modules like this to a namespace, because the `@memberof` tag does not work in combination with the `@module` tag.